KsApi/ServiceType.swift

```diff
@@ -106,6 +100,16 @@ public protocol ServiceType {
   /// Fetch friend stats.
   func fetchFriendStats() -> SignalProducer<FriendStatsEnvelope, ErrorEnvelope>
 
+  /// Fetch generic objects using graphQL.
+  func fetchGraph<A: Swift.Decodable>(query: NonEmptySet<Query>) -> SignalProducer<A, GraphError>
```

Comment:
We could get around the `fetchGraph` prefixes by having a `nested` protocol type, similar to how we do `inputs` and `outputs` on view models. Not a big deal but might make it read better as `apiService.graph.fetchCategories`, makes it seems less as if `GraphCategory` is a thing 😄

Reply:
This function actually will be removed since after some refactoring and code cleanup, it's not necessary anymore, because we can simply call the private `fetch` function like:  
`public func fetchGraphCategories(query: NonEmptySet<Query>) -> SignalProducer<RootCategoriesEnvelope, GraphError> {
    return fetch(query: query)
  }`